source/game/system/map/MapdataGeoObj.hh

```diff
@@ -0,0 +1,51 @@
+#pragma once
+
+#include "game/system/map/MapdataAccessorBase.hh"
+
+#include <egg/math/Vector.hh>
+
+namespace System {
+
+class MapdataGeoObj {
+public:
+    static constexpr int N_SETTINGS = 8;
```

Comment:
@vabold Uncertain what to do here... First it should probably be `size_t`, it should not be public (but I understand `SData` being dependent on it)...

Reply:
Because it instantiates an `std::array`, it should be `size_t`.

Recall what we did in `KartBoost` to solve a similar problem. We instantiated the count early to allow for member initialization. The same solution can be applied here, due to the fact that `SData` is big endian and thus unreliable, meaning its contents should be copied into the `MapdataGeoObj` during its construction (see `MapdataStartPoint` for further context).